src/GitLabApiClient/Internal/Queries/GroupsProjectsQueryBuilder.cs

```diff
@@ -75,7 +80,7 @@ private static string GetOrderQueryValue(GroupsProjectsOrder order)
                 case GroupsProjectsOrder.LastiActivityAt:
                     return "last_activity_at";
                 default:
-                    return "created_at";
```

Comment:
same here we need CreatedAt condition

Reply:
createdAt also already added in  the switch statement